sys/suit/handlers_command_seq.c

```diff
@@ -314,8 +373,8 @@ static int _dtv_verify_image_match(suit_manifest_t *manifest, int key,
      * riotboot_flashwrite_verify_sha256() is only interested in the 32b digest,
      * so shift the pointer accordingly.
      */
-    res = riotboot_flashwrite_verify_sha256(digest + 4,
```

Comment:
Now that comment needs an update

Reply:
Removed the comment, it doesn't add anything and I consider the function here not complex enough to warrant a comment :)

Reply:
Can you explain this change?

Reply:
Of course, the digest as contained in the SUIT manifest lives in a [SUIT digest container](https://tools.ietf.org/html/draft-ietf-suit-manifest-09#section-10). A reference to the digest container is extracted [here](https://github.com/RIOT-OS/RIOT/pull/14436/files#diff-66b52a93d9ad7c336bcc3c9aa5fdd47eR361) and the digest itself is retrieved [here](https://github.com/RIOT-OS/RIOT/pull/14436/files#diff-66b52a93d9ad7c336bcc3c9aa5fdd47eR365).

Where previously the digest container itself was referenced, and where it was assumed that the digest would start at 4 bytes into this container, the code now properly retrieves a pointer to the digest itself by parsing the container. This removes the need of the hardcoded offset, as the `const uint8_t *digest` is now a reference to the digest bytes.